rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -100,6 +101,10 @@ LinearSimple. ReceiptLinearImpl ::= [LinearBind] ;
 -- for (x <- a; y <- b if *x)
 -- LinearCond.   Linear ::= [LinearBind] "if" Proc ;
 
+-- Single Declaration
+DeclarationImpl. Decl ::= Proc "=" Proc ;
```

Comment:
this is far too permissive. Since
```
let p1 = e1, ..., pn = en in { Q }

//is syntax sugar for

match e1 {
  p1 => {
    match e2 {
      p2 => {
...
        match en {
          pn => Q
        }
...
      }
    }
  }
}
```

this should be sth like
`DeclarationImpl. Decl ::= Proc13 "=" Proc4 ;`

(it should use the same parts of grammar the match uses)